samples/resource-server-sample/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+	id 'org.springframework.boot' version '2.2.2.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+	id 'java'
+}
+
+group = 'sample'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.security:spring-security-oauth2-resource-server:5.3.1.RELEASE'
+	implementation 'org.springframework.security:spring-security-oauth2-jose:5.3.1.RELEASE'
+	implementation 'org.springframework.security:spring-security-config:5.3.1.RELEASE'
+	
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+}
+
+test {
+	useJUnitPlatform()
```

Comment:
Should we use JUnit4 or JUnit5 for testing?

Reply:
I changed it to use junit 5 api, as that was also used in the parent